src/components/displayDocuments/displayDocuments.test.tsx

```diff
@@ -40,63 +42,125 @@ const sampleDocuments: Document[] = [
     uri: "test6",
     imageSrc: "",
     tags: [Tag.LICENCE, Tag.STORABLE]
+  },
+  {
+    title: "test7",
+    uri: "test7",
+    imageSrc: "",
+    tags: [Tag.HEALTH_CERTS]
   }
 ];
 
 describe("displayDocuments", () => {
-  it("should show all documents, when 'All' button is pressed", () => {
+  it("should show all documents, when in home route", () => {
     expect.assertions(1);
-    const { getByText, getAllByTestId } = render(<DisplayDocuments documents={sampleDocuments} />);
-    const allButton = getByText("All");
-    fireEvent.click(allButton);
-    const displayCard = getAllByTestId("display-card");
-    expect(displayCard).toHaveLength(sampleDocuments.length);
+    const history = createMemoryHistory();
+    render(
+      <Router history={history}>
+        <Route path={`/`}>
+          <DisplayDocuments documents={sampleDocuments} />
+        </Route>
+      </Router>
+    );
+    expect(screen.getAllByTestId("display-card")).toHaveLength(sampleDocuments.length);
   });
 
-  it("should show only TradeTrust documents, when 'TradeTrust' button is pressed", () => {
+  it("should show only TradeTrust documents, when in 'trade-trust' route", () => {
     expect.assertions(1);
-    const { getByText, getAllByTestId } = render(<DisplayDocuments documents={sampleDocuments} />);
-    const tradeTrustButton = getByText("TradeTrust");
-    fireEvent.click(tradeTrustButton);
-    const displayCard = getAllByTestId("display-card");
-    expect(displayCard).toHaveLength(1);
+    const history = createMemoryHistory();
+    history.push("/tag/trade-trust");
```

Comment:
any reason to not use query param instead ? 

`?tag=trade-trust`

Reply:
um no particular reason actually. i always strive to construct pretty urls instead of query params:

1. easy to rmb
2. looks nice
3. seo friendly (although i not sure if has any real benefits on SPAs, the last i rmb crawlers don't really manage well with js-based SPAs)

im ok to do query param way also if the situation doesn't really works out

Reply:
ok then let's keep the url